beacon_node/network/src/sync/peer_sync_info.rs

```diff
@@ -86,8 +86,8 @@ impl PeerSyncInfo {
             // that we are within SLOT_IMPORT_TOLERANCE of our two heads
             if (self.head_slot >= remote.head_slot
                 && self.head_slot.sub(remote.head_slot).as_usize() <= SLOT_IMPORT_TOLERANCE)
-                || (self.head_slot < remote.head_slot)
-                    && remote.head_slot.sub(self.head_slot).as_usize() <= SLOT_IMPORT_TOLERANCE
+                || (self.head_slot < remote.head_slot
+                    && remote.head_slot.sub(self.head_slot).as_usize() <= SLOT_IMPORT_TOLERANCE)
```

Comment:
This is clearer, but equivalent, because `&&` binds tighter than `||` -- the parens do nothing.